incubator/command-log/src/main/java/io/aklivity/zilla/runtime/command/log/internal/LoggableStream.java

```diff
@@ -963,7 +963,7 @@ private String asString(
             final OctetsFW headerName = header.name();
             final String formattedHeaderName = headerName.buffer().getStringWithoutLengthUtf8(
                 headerName.offset(), headerName.sizeof());
-            formatted = String.format("header[%s=[%d]]", formattedHeaderName, header.valueLen());
+            formatted = String.format("header[%s=%s]", formattedHeaderName, asString(header.value()));
```

Comment:
We cannot assume that Kafka header values are strings.

Reply:
Yeah, left over I was debugging